sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -260,20 +260,21 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 	isGlobalPullSecret := secret.GetNamespace() == openshiftConfigNamespace && secret.GetName() == openshiftConfigPullSecret
 
 	newIntegrationSet := set.NewStringSet()
-	for registry, dce := range dockerConfig {
+	for registryAddr, dce := range dockerConfig {
+		registryAddr = strings.TrimSpace(registryAddr)
```

Comment:
I think we should log a warning if we are actually trimming spaces here. If the user has a secret with spaces we do our best effort to remove them but at the end of the day this is a misconfiguration on their side so I think we should log it.

Reply:
Added that in [8c48d71](https://github.com/stackrox/stackrox/pull/11668/commits/8c48d719796f45beb375f8a0a49bdbed0e389e6c)